src/pages/advance/clerk-idp.mdx

```diff
@@ -0,0 +1,69 @@
+import { Callout } from "nextra-theme-docs";
+import { Images } from "@/components/Images";
+
+# Use Clerk as an OAuth 2.0 Identity Provider 
+
+Learn how to use Clerk as an Identity provider to facilitate Single Sign-On (SSO) with other applications that support the OAuth2 protocol. 
+
+<Images
+  width={496}
+  height={564}
+  src="/images/advance/clerk-idp-flow.svg"
+  alt="Clerk SSO IDP Flow"
+/>
+
+
+## Step 1: Retrieve SSO Provider Callback URL
+
+From your SSO provider, retrieve the **Callback URL**. This is where your users will be redirected to after a successful authentication and is required to successfully create your OAuth application in the next step.
+
+
+## Step 2: Create an OAuth Application
+
+Next, you need to create an OAuth Application in Clerk via the [Backend API](https://clerk.com/docs/reference/backend-api/tag/OAuth-Applications).
+
+```sh copy filename="POST /v1/oauth_applications"
+curl 
+ -X POST https://api.clerk.com/v1/oauth_applications \
+ -H "Authorization: Bearer <CLERK_SECRET_KEY>"  \
+ -H "Content-Type: application/json" \
+ -D {"callback_url":"https://oauth-client.com/oauth2/callback", "name": "oauth_app_1"}
+```
+
+Clerk creates an OAuth application and returns it's configurations back to you.
+
+
+```json copy filename="RESPONSE"
+{
+    "object":"oauth_application",
+    "id":"oa_2O4BCh3zUONvWlZHtBXv6s6tm7u",
+    "instance_id":"ins_2O4Ak02T4fDmKKv6tK5h0WJ8ou8",
+    "name":"oauth_app_1",
+    "client_id":"d9g4CT4WYiCBm7EU",
+    "client_secret":"VVgbT7i6sPo7sTljq2zj12fjmg0jPL5k",
+    "scopes":"profile email",
+    "callback_url":"https://oauth-client.com/oauth2/callback",
+    "authorize_url":"https://clerk.your-domain.com/oauth/authorize",
+    "token_fetch_url":"https://clerk.your-domain.com/oauth/token",
+    "user_info_url":"https://clerk.your-domain.com/oauth/userinfo",
+    "created_at":1680809847940,
+    "updated_at":1680810135145
+}
+```
+
+<Callout type="warning" emoji="⚠️">
+  For security reasons, we don’t store your Client Secret and cannot show it to you again, so we recommend you download the secret and store it someplace secure.
+</Callout>
+
+
+## Step 3: Configure your SSO Provider
+
+Now that you have setup an OAuth application in Clerk, you will need to configure these settings in your SSO provider.
+
+* **Client ID**: Public identifier of your application
+* **Client Secret**: Confidential secret used to authenticate your OAuth application
+* **Authorization URL**: Used by the SSO provider to authorization your OAuth application
```

Comment:
In particular, the authorization url is how the application requests authorization from the user. A better working may be `Used by the application/OAuth client (term is TBD here) to request authorization from your user.